operation/fhir-operation-erase/src/main/java/com/ibm/fhir/operation/erase/adapter/ResourceEraseRecordAdapter.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * (C) Copyright IBM Corp. 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.operation.erase.adapter;
+
+import static com.ibm.fhir.model.type.String.string;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+import com.ibm.fhir.model.resource.Parameters;
+import com.ibm.fhir.model.resource.Parameters.Parameter;
+import com.ibm.fhir.persistence.ResourceEraseRecord;
+import com.ibm.fhir.persistence.ResourceEraseRecord.Status;
+import com.ibm.fhir.persistence.erase.EraseDTO;
+import com.ibm.fhir.server.util.FHIROperationUtil;
+
+/**
+ * Adapts the ResourceEraseRecord and EraseBean to a Parameters object
```

Comment:
during our review session I think you mentioned that you want to wipe out references to EraseBean...just double checking we want this (i'm not sure what it is)

Reply:
Yes - fixed. and thank you